frontend/src/app/shared/components/cluster-table/renderer/cluster-actions-renderer.component.ts

```diff
@@ -25,8 +25,8 @@ import { Cluster } from "src/app/modules/cluster/models/cluster";
       <div class="flex">
           <button
                 routerLink="/clusters/detail/{{ data.id }}"
-                class="btn">
-            <svg-icon src="assets/icons/outline/arrow-sm-right.svg" [svgClass]="'h-5 w-5'"> </svg-icon>
+                class="btn btn-primary">
```

Comment:
You are missing the changes to the `style.scss` file mentioned in [this comment](https://github.com/ArubaKube/liqo-dashboard/pull/8#discussion_r2265914498)